packages/dd-trace/test/appsec/iast/vulnerability-formatter/resources/evidence-redaction-suite.json

```
@@ -2679,6 +2679,57 @@
           }
         ]
       }
+    },
+    {
+      "type": "VULNERABILITIES",
+      "description": "SQLi exploited",
+      "input": [
+        {
+          "type": "SQL_INJECTION",
+          "evidence": {
+            "value": "SELECT * FROM Users WHERE email = '' OR TRUE --' AND password = '81dc9bdb52d04dc20036dbd8313ed055' AND deletedAt IS NULL",
+            "ranges": [
+              {
+                "start": 35,
+                "end": 47,
+                "iinfo": {
+                  "type": "http.request.parameter",
+                  "parameterName": "email",
+                  "parameterValue": "' OR TRUE --"
+                }
+              }
+            ]
+          }
+        }
+      ],
+      "expected": {
+        "sources": [
+          {
+            "origin": "http.request.parameter",
+            "name": "email",
+            "value": "' OR TRUE --"
+          }
+        ],
+        "vulnerabilities": [
+          {
+            "type": "SQL_INJECTION",
+            "evidence": {
+              "valueParts": [
+                {
+                  "value": "SELECT * FROM Users WHERE email = '"
+                },
```

Probably we will change this to something like this:
```suggestion
                },
                {
                  "redacted": true
                },
```